angular.js

```diff
@@ -9,6 +9,23 @@ module.exports = {
     './overrides.js', // this should always be last because it's our common rules and contains prettier
   ],
   rules: {
+    'no-restricted-imports': [
```

Comment:
The current config in NgApps (and any other repos) will override this config because it has higher precedence so you're also going to need to drop it from the other repos.